Elements.css

```diff
@@ -1,117 +1,141 @@
 #outside_container {
-  position: fixed;
-  top: 0;
-  left: 0;
-  width: 100%;
-  height: 100%;
-  background: url(background.jpg);
-  z-index: -1;
+    position: fixed;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    background: url(background.jpg);
+    z-index: -1;
 }
 
 #container {
-  top: 0;
-  left: 0;
-  width: 100%;
-  height: 100%;
-
-  z-index: 1;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    z-index: 1;
+    margin-bottom: 50px;
+    overflow: auto;
 }
 
 .container_2 {
-  top: 0;
-  left: 0;
-  width: 100%;
-  height: 100%;
-
-  z-index: 1;
+    border: 2px solid #000000c5;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    z-index: 1;
+    background: #c0d698c5;
+    overflow: auto;
+    margin-bottom: 50px;
 }
 
 #avatar_container {
-  position: relative;
-  top: 30%;
-  left: 50%;
-  transform: translate(-50%, -70%);
-  width: 50%;
-  height: 20%;
+    position: relative;
+    top: 30%;
+    left: 50%;
+    transform: translate(-50%, -70%);
+    width: 50%;
+    height: 20%;
 }
 
 #photo {
-  position: relative;
-  top: 50%;
-  left: 50%;
-  width: 200px;
-  height: 200px;
-  border-radius: 50%;
-  transform: translate(-50%, -50%);
-  background: url(photo.jpeg);
-  background-size: cover;
-  border: 2px;
-  z-index: 2;
+    position: relative;
+    top: 50%;
+    left: 50%;
+    width: 200px;
+    height: 200px;
+    border-radius: 50%;
+    transform: translate(-50%, -50%);
+    background: url(photo.jpeg);
+    background-size: cover;
+    border: 2px;
+    z-index: 2;
 }
 
 .info {
-  text-align: center;
-
-  margin-top: 10px;
-  color: #0b0329;
-  margin-bottom: 70px;
-  font: bold italic 110% serif;
+    text-align: center;
+    margin-top: 10px;
+    color: #0b0329;
+    margin-bottom: 70px;
+    font: bold italic 110% serif;
```

Comment:
It's better to setup font properties separately. font-size, font-weight, font-family, font-style

Reply:
fixed